### DISCUSSION TITLE
How to remove the `__typename` field?

### DISCUSSION THREAD

**Discussion by cdwmhcc** [nuxt-community/apollo-module#462] [Q&A] [answered]
How to remove the `__typename` field?

**Comment by cdwmhcc** [comment, marked as answer]
```
addTypename: false
```

**Reply by slavanossar** [reply]
Just be aware that this could affect Apollo's automatic caching, since it creates unique identifiers for cache entries with format `__typename:id`.

For example, the following query response

```
{
  __typename: 'Article',
  id: 'abc123'
}
```

would be cached as `Article:abc123`
